react/training-project/src/app/components/Game/index.js

```diff
@@ -1,21 +1,75 @@
-import React from 'react';
+import React, { Component } from 'react';
 
 import Board from '../Board';
 
 import style from './styles.scss';
 
-function Game() {
-  return (
-    <div className={style.game}>
-      <div className={style.gameBoard}>
-        <Board />
-      </div>
-      <div className={style.gameInfo}>
-        <div>{/* status */}</div>
-        <ol>{/* TODO */}</ol>
+class Game extends Component {
+  state = {
+    history: [{ squares: Array(9).fill(null) }],
+    stepNumber: 0,
+    xIsNext: true
+  };
+
+  calculateWinner = squares => {
+    const lines = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
```

Comment:
I would extract it or you'll be creating this array every time you call `calculateWinner`.